### DISCUSSION TITLE
Enable login on more than one device

### DISCUSSION THREAD

**Discussion by kulfuerst** [Authenticator-Extension/Authenticator#858] [Ideas]
I need to log in on 3 separate devices using the same account. Currently, I cannot connect the authenticator to the same account on more than one device. Would it be possible for such a feature to be implemented? Thank you!

**Comment by Sneezry** [comment]
What kind of device are you using?

**Comment by kulfuerst** [comment]
Hi,
It's two Windows 10 PCs and one MacBook.


Zhe Li ***@***.***> schrieb am Do., 17. Feb. 2022, 11:43:

**Reply by Sneezry** [reply]
The data will be synced among devices if you login browser account and enable browser-sync
